standalone-e2e-test/README.MD

```diff
@@ -0,0 +1,10 @@
+mvn clean install -f standalone-e2e-test/pom.xml
```

Comment:
Instead of putting an additional maven modul inside the existing one, maybe you can try to refactor the whole project to a real multi module maven project: 
* Theory: https://www.baeldung.com/maven-multi-module
* In practice: https://mkyong.com/maven/maven-how-to-create-a-multi-module-project/

Reply:
Changed. Thanks for the PAIN :D